lib/apimatic-core/utilities/constants.rb

```diff
@@ -0,0 +1,11 @@
+module CoreLibrary
+  CONTENT_TYPE_HEADER = 'content-type'.freeze
+  ACCEPT_HEADER = 'accept'.freeze
+  CONTENT_LENGTH_HEADER = 'content-length'.freeze
+  AUTHORIZATION_HEADER = 'authorization'.freeze
+  FORM_URLENCODED_CONTENT_TYPE = 'application/x-www-form-urlencoded'.freeze
+  JSON_CONTENT_TYPE = 'application/json'.freeze
+  TEXT_CONTENT_TYPE = 'text/plain; charset=utf-8'.freeze
+  XML_CONTENT_TYPE = 'application/xml'.freeze
+  TEST_TOKEN = 'MyDuMmYtOkEn'.freeze
```

Comment:
Are we using this test constant in the source of the core library? If not, tests specific constants should not be added here. Also, you can extract the following magic strings from SdkLogger class, for example

```Ruby
      @logger.log(@log_level, 'Request {method} {url} {content_type}', {
                    'method' => request.http_method,
                    'url' => url,
                    'content_type' => content_type_header
                  })
```

Reply:
what constants are you referring here, This change was made to cater for the above [comment](https://github.com/apimatic/core-lib-ruby/pull/37#discussion_r1593985831)